packs/nitropack.js

```diff
@@ -0,0 +1,41 @@
+  /** Additional description of a Lighthouse audit for a third-party framework called `NitroPack`. This is displayed after a user expands the section to see more. No character length limits. `Remove render-blocking resources`, `Delayed Scripts`, `Font Subsetting`, `Optimized CSS Delivery` and `Reduce Unused CSS` are names of features and become links to additional documentation.*/
+  'render-blocking-resources': 'Enable [`Remove render-blocking resources`](https://support.nitropack.io/hc/en-us/articles/13820893500049-How-to-Deal-with-Render-Blocking-Resources-in-NitroPack) and [`Delayed Scripts`](https://support.nitropack.io/hc/en-us/articles/1500002600942-Delayed-Scripts) in NitroPack for faster initial load times. For even better results, activate [`Font Subsetting`](https://support.nitropack.io/hc/en-us/articles/10172273306769), [`Optimize CSS Delivery`](https://support.nitropack.io/hc/en-us/articles/16547344670353) and [`Reduce Unused CSS`](https://support.nitropack.io/hc/en-us/articles/360020418457-Reduce-Unused-CSS) – these features will optimize your CSS and font files so that they don’t block the rendering of your page.',
+  /** Additional description of a Lighthouse audit for a third-party framework called `NitroPack`. This is displayed after a user expands the section to see more. No character length limits. `Minify resources` is the name of a feature and becomes link text to additional documentation.*/
+  'unminified-css': 'Enable [`Minify resources`](https://support.nitropack.io/hc/en-us/articles/360061059394-Minify-Resources) in your Caching settings to reduce the size of your CSS, HTML, and JavaScript files for faster load times. This will remove any spaces or comments from your files without changing it\'s functionality.',
+  /** Additional description of a Lighthouse audit for a third-party framework called `NitroPack`. This is displayed after a user expands the section to see more. No character length limits. `Minify resources` is the name of a feature and becomes link text to additional documentation.*/
+  'unminified-javascript': 'Enable [`Minify resources`](https://support.nitropack.io/hc/en-us/articles/360061059394-Minify-Resources) in your Caching settings to reduce the size of your JS, HTML, and CSS files for faster load times. This will remove any spaces or comments from your files without changing it\'s functionality.',
```

Comment:
Is "Caching settings" the name of a feature in NitroPack?

```suggestion
  'unminified-javascript': 'Enable [`Minify resources`](https://support.nitropack.io/hc/en-us/articles/360061059394-Minify-Resources) in your Caching settings to reduce the size of your JS, HTML, and CSS files for faster load times.',
```